src/java.base/share/classes/javax/net/ssl/SSLParameters.java

```diff
@@ -696,4 +698,94 @@ public void setApplicationProtocols(String[] protocols) {
         }
         applicationProtocols = tempProtocols;
     }
+
+    /**
+     * Returns a prioritized array of signature scheme names that can be used
+     * over the SSL/TLS/DTLS protocols.
+     * <p>
+     * Note that the standard list of signature scheme names are defined in
+     * the <a href=
+     * "{@docRoot}/../specs/security/standard-names.html#signature-schemes">
+     * Signature Schemes</a> section of the Java Security Standard Algorithm
+     * Names Specification.  Providers may support signature schemes not defined
+     * in this list or may not use the recommended name for a certain
+     * signature scheme.
+     * <p>
```

Comment:
Regarding my previous comment about providers not supporting these new methods, I think something like the following might suffice:

```
@apiNote Note that a provider may not have been updated to support this method and in that case may return null instead of the default signature schemes for pre-populated objects.
@implNote The SunJSSE provider supports this method.
```

Reply:
It looks like a typo. Replaced "pre-populated objects" with "connection populated objects", and added to the spec.